apps/website/docs/setup/_guilded-setup.mdx

```diff
@@ -1,9 +1,12 @@
 ## Guilded
 
-Codeize is gonna write this part cause I have no fucking clue
+To get your Guilded API key
 
-so ttue
+First, head to your server that you want to be "home base" for your bot.
+Go into the drop down menu -> Manage bots, and in the window that opens click "Create Bot"
 
-wedfuohwefuohefwujihwerfhuiqwdefedhuoqwqwedfhuiohio
+### Your bots Token
+After the bot is created go into the API tab and click "Generate Token" this will show you your api key that wont get shown again.
```

Comment:
```suggestion
After the bot is created, go into the API tab and click "Generate Token". This will show you your API key which will not be shown again.
```